app/test_main.py

```diff
@@ -1,3 +1,39 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, goals",
+    [pytest.param(14, 14, [0, 0],
+                  id="should return first 15 dog and cat years"),
+     pytest.param(28, 29, [3, 3],
+                  id="should return every 5 next dog and cat years"),
+     pytest.param(27, 28, [2, 2],
+                  id="should return every 5 next dog and cat years"),
+     pytest.param(23, 23, [1, 1],
+                  id="should return next 9 dog and cat years"),
```

Comment:
The expected result `[1, 1]` for `cat_age=23` and `dog_age=23` is correct according to the task description, as both ages are within the first 24 years, which should result in 1 human year for both.